.gitignore

```diff
@@ -1,3 +1,6 @@
 .qmake.stash
 Makefile
 webui/*
+webui.pro.user
+build.sh
+libwebui.so*
```

Comment:
Were these files created when you built the app? Not sure if this will be required

Reply:
- webui.pro.user cames from the IDE i think,
- build.sh came from me and contains the build script with (qmake CONFIG+="rs_jsonapi"; make), we can add it to project, if ok
- libwebui.so, libwebui.so.1,libwebui.so.1.0 and libwebui.so.1.0.0  produced by the build commands, if executed in project directory